torchrl/objectives/value/functional.py

```diff
@@ -403,12 +395,13 @@ def td1_advantage_estimate(
         )
     if not state_value.shape == next_state_value.shape:
         raise RuntimeError("shape of state_value and next_state_value must match")
-    returns = td1_return_estimate(gamma, next_state_value, reward, done, rolling_gamma)
+    returns = td1_return_estimate(
+        gamma, next_state_value, reward, done, rolling_gamma, time_dim=time_dim
+    )
     advantage = returns - state_value
     return advantage
 
 
-@_transpose_time
```

Comment:
You mean this one?

Reply:
The problem is that if I readd this line and then inside the function we do
```
  return vec_td_lambda_return_estimate(
        gamma=gamma,
        next_state_value=next_state_value,
        reward=reward,
        done=done,
        rolling_gamma=rolling_gamma,
        lmbda=1,
        time_dim=time_dim, # this line
    )
```
than it creates a bug because time_dim is not the right one anymore.

On the other hand , if we readd it and remove ` time_dim=time_dim` (like before this pr), we are still error prone because we are relying on the default_dim dim which is something we want to deprecate.